sdk/core/core-http/src/util/xml.browser.ts

```diff
@@ -33,14 +33,30 @@ function getSerializer(): XMLSerializer {
   return cachedSerializer;
 }
 
+// Policy to make our code Trusted Types compliant.
+//   https://github.com/w3c/webappsec-trusted-types
+// We are calling DOMParser.parseFromString() to parse XML payload from Azure services.
+// The parsed DOM object is not exposed to outside. Scripts are disabled when parsing
+// according to the spec.  There are no HTML/XSS security concerns on the usage of
+// parseFromString() here.
+let ttPolicy: Pick<TrustedTypePolicy, "createHTML"> | undefined;
+if (typeof self.trustedTypes !== "undefined") {
+  ttPolicy = self.trustedTypes.createPolicy("@azure/core-http#xml.browser", {
```

Comment:
I thought about making `ttPolicy` harder for bad guys to use, but we can't really hide things in JavaScript?

Reply:
I mean this is stuck inside a module scope and not exported, so I don't think anyone could easily access it?

Reply:
You are right.